routes/infrastructure/construct.py

```diff
@@ -0,0 +1,96 @@
+"""CDK Construct for a Cloudfront Distribution."""
+from typing import Optional
+from urllib.parse import urlparse
+
+from aws_cdk import CfnOutput, Duration, Stack
+from aws_cdk import aws_certificatemanager as certificatemanager
+from aws_cdk import aws_cloudfront as cf
+from aws_cdk import aws_cloudfront_origins as origins
+from aws_cdk import aws_s3 as s3
+from constructs import Construct
+
+from .config import veda_route_settings
+
+
+class CloudfrontDistributionConstruct(Construct):
+    """CDK Construct for a Cloudfront Distribution."""
+
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        raster_api_id: str,
+        stac_api_id: str,
+        region: Optional[str],
+        **kwargs,
+    ) -> None:
+        """."""
+        super().__init__(scope, construct_id)
+
+        stack_name = Stack.of(self).stack_name
+
+        if veda_route_settings.cloudfront:
+            s3Bucket = s3.Bucket.from_bucket_name(
+                self,
+                "stac-browser-bucket",
+                bucket_name=veda_route_settings.stac_browser_bucket,
+            )
+
+            no_cache_policy = cf.CachePolicy(
+                self,
+                "no-cache-policy",
+                default_ttl=Duration.seconds(0),
+                min_ttl=Duration.seconds(0),
+                max_ttl=Duration.seconds(0),
+            )
```

Comment:
there's already no cache policy defined in cdk,

```python
cf.CachePolicy.CACHING_DISABLED
```

Reply:
Done